stress/shell/src/main/java/alluxio/stress/cli/GetPinnedFileIdsBench.java

```diff
@@ -86,6 +86,12 @@ public void prepare() throws Exception {
     // So including that in the log can help associate the log to the run
     LOG.info("Task ID is {}", mBaseParameters.mId);
 
+    // the preparation is done by the invoking client
+    // so skip preparation when running in job worker
+    if (mBaseParameters.mDistributed) {
+      return;
```

Comment:
You may want to add some logs so it's easier to track what happened and what not

Reply:
Added logs